src/frontend/komponenter/Fagsak/Vilkår/BehandleVilkår.tsx

```diff
@@ -33,7 +33,7 @@ const BehandleVilkår: React.FunctionComponent<IProps> = ({ fagsak }) => {
     );
 
     React.useEffect(() => {
-        if (aktivBehandling && aktivBehandling.samletVilkårResultat) {
+        if (aktivBehandling && aktivBehandling.samletVilkårResultat && aktivBehandling.samletVilkårResultat.length > 0) {
```

Comment:
Is it so that a behandling always has a `samletVilkårResultat`-list (but sometimes empty), so we just can write `aktivBehandling && aktivBehandling.samletVilkårResultat.length > 0` ?

Reply:
Correct!